src/doc_scan_service/session/retrieve/doc.scan.session.js

```diff
@@ -0,0 +1,133 @@
+'use strict';
+
+const Validation = require('../../../yoti_common/validation');
+const ResourceContainer = require('./resource.container');
+const CheckResponse = require('./check.response');
+const AuthenticityCheckResponse = require('./authenticity.check.response');
+const FaceMatchCheckResponse = require('./face.match.check.response');
+const TextDataCheckResponse = require('./text.data.check.response');
+const ZoomLivenessCheckResponse = require('./zoom.liveness.check.response');
+const DocScanConstants = require('../../doc.scan.constants');
+
+class DocScanSession {
```

Comment:
This should be called `GetSessionResult` to bring it in-line with already released Java SDK

Reply:
Renamed in 4966c52